docusaurus/website/docusaurus.config.base.js

```diff
@@ -193,6 +193,14 @@ const plugins = [
           from: ['/create-a-plugin/extend-a-plugin/extend-configurations'],
           to: '/get-started/set-up-development-environment#extend-configurations',
         },
+        {
+          from: ['/migration-guides/update-create-plugin-versions'],
+          to: '/reference/cli-commands/#update-plugin-tooling',
```

Comment:
Does this work? From what I know the docusaurus plugin creates html pages for "from" with a client side redirect to "to".